src/views/Homepage/About/texts.tsx

```diff
@@ -13,14 +13,13 @@ const TEXTS: Record<string, any> = {
   en: {
     title: "About Matters Lab",
     content1:
-      "Matters Lab was established in 2018, with the mission to create a freer and fairer creator ecosystem " +
-      "through the next evolution of the Internet, Web3. Our initiatives include using decentralization tools to " +
-      "protect digital rights, design a self-governing system and community, and establish a creator economy model.",
+      "Matters Lab was established in 2018, with the mission to create a freer and fairer creator ecosystem through " +
+      "the next evolution of the Internet, Web3. Our initiatives include using decentralization tools to protect digital " +
+      "rights, design a self-governing system and community, and invent models for a new creator economy.",
```

Comment:
for very long text, you may use jsx syntax here:

```jsx
content1: <>
  Matters Lab was established in 2018, with the mission to create a freer and fairer creator ecosystem through
  the next evolution of the Internet, Web3. Our initiatives include using decentralization tools to protect digital
  rights, design a self-governing system and community, and invent models for a new creator economy.
</>
...
```

this works better if some text needs to embed other HTML tags, like `<b>` `<a>` ...